packages/common/src/index.ts

```diff
@@ -623,6 +623,20 @@ export type OrganizationUser = Pick<
     'userUuid' | 'firstName' | 'lastName' | 'email'
 >;
 
+export enum OrganizationMemberRole {
+    VIEWER = 'viewer',
+    EDITOR = 'editor',
+    ADMIN = 'admin',
+}
+export type OrganizationMemberProfile = {
```

Comment:
Fyi we already have a `OrganizationUser` type but we could replace it with this one.